### PR TITLE
Fix: Web Clothing Butcher Issues

### LINKED ISSUE #33117: Spooder web boots are too cheap
## Description
You can get 200% cloth for your silk by making them into boots before cutting into cloth as making cloth from silk is twice as expensive.

**Reproduction**
1. Make silk
2. Make boots
3. Make cloth, put on ground
4. Cut boots, compare.

### PR DESCRIPTION
**Pull request by thetolbean**
## About the PR
Corrects issues related to arachnid crafting using silk.

1. Web coat is now butcherable
2. Web coat and boot now butcher down to their crafting cost in silk
    - Boot previously broke down into cloth

## Why / Balance
Fix #33117

## Technical details
Adds butcherable component to web boots and coat

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl:
- fix: The web coat and boots can now be broken down into silk.

### CODE REVIEW

**Review by PursuitInAshes — APPROVED** [review]

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml [hunk 1]
```diff
@@ -853,6 +853,11 @@
   - type: Construction
     graph: WebObjects
     node: coat
+  - type: Butcherable
+    butcheringType: Knife
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml:857]:
```suggestion
```
that's already the default value, so we can remove it

File: Resources/Prototypes/Entities/Clothing/Shoes/boots.yml [hunk 2]
```diff
@@ -187,6 +187,11 @@
         reagents:
         - ReagentId: Fiber
           Quantity: 10
+  - type: Butcherable
+    butcheringType: Knife
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Clothing/Shoes/boots.yml:191]:
```suggestion
```
same

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml (hunk 1 above)
Comment (thetolbean) [reply Resources/Prototypes/Entities/Clothing/OuterClothing/wintercoats.yml]: Removed

File: Resources/Prototypes/Entities/Clothing/Shoes/boots.yml (hunk 2 above)
Comment (thetolbean) [reply Resources/Prototypes/Entities/Clothing/Shoes/boots.yml]: Removed

**Comment by thetolbean** [comment]
Unrelated test fail

**Review by slarticodefast — APPROVED** [review]

**Comment by slarticodefast** [comment]
Tested and works as intended. Thanks for the PR!

**Comment by chromiumboy** [comment]
Thanks for the fix! I've tweaked the change log slightly for clarity, I hope you don't mind

**Pull request merged**